File: scripts/fix_wrangler.py
import re

with open('cloudflare-backend/wrangler.toml', 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace('[[r2_buckets]]', '# [[r2_buckets]]')
content = content.replace('binding = "BUCKET"', '# binding = "BUCKET"')
content = content.replace('bucket_name = "techfix-images"', '# bucket_name = "techfix-images"')

with open('cloudflare-backend/wrangler.toml', 'w', encoding='utf-8') as f:
    f.write(content)

File: scripts/fix_wrangler.js
const fs = require("fs");
let code = fs.readFileSync("cloudflare-backend/wrangler.toml", "utf8");
code = code.replace(/STRIPE_SECRET_KEY = "sk_test_.*"/, "STRIPE_SECRET_KEY = \"\" # Set securely via wrangler secret put");
fs.writeFileSync("cloudflare-backend/wrangler.toml", code);

